app/views/templates/track.jst

```diff
@@ -1,24 +1,72 @@
-<td class="number"><%- number %></td>
-<td class="title"><%- title %></td>
-<td><%- artist %></td>
-<td><%- album %>
+<div class="number-column-cell" role="gridcell">
+    <%- number %>
+    <svg>
+        <use xlink:href="<%= require('../../assets/icons/speaker-sm.svg') %>"/>
+    </svg>
+</div>
+<div class="song-column-cell" role="gridcell"><%- title %></div>
+<div class="artist-column-cell" role="gridcell"><%- artist %></div>
+<div class="album-column-cell" role="gridcell"><%- album %>
     <div class="actions">
+        <% if (type != 'upNext') { %>
         <button id="add-to-upnext">
             <p class="help">Add To Up Next</p>
             <svg>
                 <use xlink:href="<%= require('../../assets/icons/add-to-playlist-sm.svg') %>"/>
             </svg>
         </button>
+        <% } else { %>
         <button id="delete-from-upnext">
             <svg>
                 <use xlink:href="<%= require('../../assets/icons/cross-sm.svg') %>"/>
             </svg>
         </button>
+        <% } %>
         <button id="menu">
             <svg>
                 <use xlink:href="<%= require('../../assets/icons/more-sm.svg') %>"/>
             </svg>
         </button>
     </div>
-</td>
-<td><%- duration %><td>
+</div>
+<div class="length-column-cell" role="gridcell"><%- duration %></div>
+<div id="popup-menu" class="popup-menu">
```

Comment:
This isn't legit: you can have an `id` more than one time in the whole document markup. As long as this template will be duplicated for each track displayed, you'll have multiple times the same `id` (this is the reason why I insist to avoid `id`s unless they're clearly needed and easy to isolate)